pkg/routing/utils.go

```diff
@@ -18,5 +18,5 @@ func parseParticipantKey(pkey string) (roomName livekit.RoomName, identity livek
 		return
 	}
 
-	return parts[0], parts[1], nil
+	return livekit.RoomName(parts[0]), parts[1], nil
```

Comment:
It will eventually. As `RoomName` is the first one I am converting to a type definition, need the cast here. The rest are all type aliases for now. Plan to make one or two a type definition at a time and add the necessary casts.